AxisCamera/vision.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+
+import ntcore
+import numpy
+from cscore import CameraServer
+import cv2
+
+
+#
+# This code will work both on a RoboRIO and on other platforms. The exact mechanism
+# to run it differs depending on whether you’re on a RoboRIO or a coprocessor
+#
+# https://robotpy.readthedocs.io/en/stable/vision/code.html
+
+
+def main():
+    # Get the Axis camera from CameraServer
+
+    camera = CameraServer.addAxisCamera("axis-camera.local")
+    # Set the resolution
```

Comment:
Everything looks good, but can you associate the comments with the lines that they're with?

```python
# Get the Axis camera from CameraServer
camera = CameraServer.addAxisCamera("axis-camera.local")

# Set the resolution
camera.setResolution(640, 480)
```

Reply:
Done with c62dc87